tests/RegexTest.php

```diff
@@ -164,7 +164,7 @@ private function doExpectException($class, $message = null)
     private function doExpectWarning($message)
     {
         if (method_exists($this, 'expectWarning')) {
-            $this->expectWarning($message);
+            $this->expectWarning();
```

Comment:
Why this change?

Reply:
PhpStan picked up that `expectWarning` doesn't take any params: https://github.com/sebastianbergmann/phpunit/blob/ee1820c53e849c5da75f26ef4c09995a4ff457cc/src/Framework/TestCase.php#L445-L448

`expectWarningMessage` does though, so I'll change it to this.

Reply:
Actually you need both:
```php
if (method_exists($this, 'expectWarning')) {
    $this->expectWarning();
    $this->expectWarningMessage($message);
}
```